frontend/www/js/omegaup/components/login/Signup.vue

```diff
@@ -106,31 +128,79 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import omegaup_Markdown from '../Markdown.vue';
 import T from '../../lang';
-
+import * as time from '../../time';
+import DatePicker from '../DatePicker.vue';
+import * as ui from '../../ui';
 @Component({
   components: {
     'omegaup-markdown': omegaup_Markdown,
+    'omegaup-datepicker': DatePicker,
   },
 })
 export default class Signup extends Vue {
   @Prop() validateRecaptcha!: boolean;
-
   T = T;
   username: string = '';
-  email: string = '';
+  email: null | string = null;
+  parentEmail: null | string = null;
   password: string = '';
   passwordConfirmation: string = '';
   recaptchaResponse: string = '';
+  birthDate: null | Date = null;
+  privacyPolicyAccepted = false;
+  get loginEmailDescriptionText(): string {
```

Comment:
```suggestion

  get loginEmailDescriptionText(): string {
```

for consistency with the rest of the code, please keep an empty line between functions.